cob_hardware_config/cob4-1/config/teleop.yaml

```diff
@@ -1,20 +1,97 @@
+#cob4-1
+##########
 # common params
 run_factor: 2.0
+joy_num_buttons: 12
+joy_num_axes: 6
+joy_num_modes: 4
+mode_switch_button: 8
+light_topic_name: /light_torso/light
+
+# axes
+axis_vx: 1
+axis_vy: 0
+axis_vz: 5
+axis_roll: 4
+axis_pitch: 3
+axis_yaw: 2
 
 # buttons
 deadman_button: 5
+safety_button: 4
+init_button: 9
+
+#mode1: Base
 run_button: 7
-base_safety_button: 4
 
-# axes
-axis_vx: 1
-axis_vy: 0
-axis_vth: 2
+#mode2: Trajectory controller
+torso_sss_default_target_button: 1
+sensorring_sss_default_target_button: 3
+head_sss_default_target_button: 3
+arm_right_sss_default_target_button: 2
+arm_left_sss_default_target_button: 0
+
+#mode3: Velocity group controller
+torso_joint1_button: 0
+sensorring_joint1_button: 1
+head_joint1_button: 2
+arm_right_joint1_button: 0
+arm_right_joint2_button: 1
+arm_right_joint3_button: 2
+arm_right_joint4_button: 3
+arm_left_joint1_button: 0
+arm_left_joint2_button: 1
+arm_left_joint3_button: 2
+arm_left_joint4_button: 3
```

Comment:
No, because it depends on your robot and your joystick configuration. It has not be joint1_button always 0, for example for the head is the joint1_button is 2.